MaterialDesignThemes.Wpf/AdornerAssist.cs

```diff
@@ -1,28 +1,44 @@
+using System;
 using System.Linq;
 using System.Windows;
 using System.Windows.Documents;
 
 namespace MaterialDesignThemes.Wpf
 {
     /// <summary>
-    /// this is a simple utility to add and remove a single adorner to an element since there is no built-in way to do that in xaml.
-    /// <a href="https://docs.microsoft.com/en-us/dotnet/framework/wpf/controls/adorners-overview">see here</a>
+    /// This is a simple utility to add and remove a single adorner to an element
+    /// since there is no built-in way to do that in xaml.
+    /// <a href="https://docs.microsoft.com/en-us/dotnet/framework/wpf/controls/adorners-overview">See here</a>
     /// </summary>
-    internal static class AdornerAssist
+    public static class AdornerAssist
```

Comment:
Do you have a need for this to be public? In general I try to leave things private until there is a need for them to be public.

Reply:
Originally the idea was to align this class with other "*Assist" classes.
Thinking further about it, i think this class is misnamed. I would propose
to keep the modifier internal and name the class "UIElementExtensions".

This more closely describes what this class actually provides and also serves as an entrypoint for further extensions of the UIElement class.